public/common/types/explorer.ts

```diff
@@ -55,6 +55,7 @@ export interface IExplorerFields {
 export interface IExplorerProps {
   tabId: string;
   query: any;
+  http: any;
```

Comment:
i think `http` is `HttpStart` in here `https://github.com/opensearch-project/OpenSearch-Dashboards/blob/1dc90f5d16316fcc49a7ef67a9e6306a407cee59/src/core/public/http/types.ts#L90:13`, might be better to reduce the number of `any` types